pkg/api/k8s_versions.go

```diff
@@ -64,206 +49,244 @@ const (
-		common.AddonManagerComponentName:  "kube-addon-manager-amd64:v8.9.1",
-		common.ClusterAutoscalerAddonName: "cluster-autoscaler:v1.12.8",
-	},
-	"1.11": {
-		"addon-resizer":                   "addon-resizer:1.8.4",
-		"metrics-server":                  "metrics-server-amd64:v0.2.1",
-		common.AddonManagerComponentName:  "kube-addon-manager-amd64:v8.9.1",
-		common.ClusterAutoscalerAddonName: "cluster-autoscaler:v1.3.9",
-	},
-	"1.10": {
-		"addon-resizer":                    "addon-resizer:1.8.4",
-		"metrics-server":                   "metrics-server-amd64:v0.2.1",
-		common.AddonManagerComponentName:   "kube-addon-manager-amd64:v8.9.1",
-		"rescheduler":                      "rescheduler:v0.3.1",
-		common.ClusterAutoscalerAddonName:  "cluster-autoscaler:v1.2.5",
-		common.NVIDIADevicePluginAddonName: "k8s-device-plugin:1.10",
-		"k8s-dns-sidecar":                  "k8s-dns-sidecar-amd64:1.14.8",
-	},
-	"1.9": {
-		"addon-resizer":                   "addon-resizer:1.8.4",
-		"metrics-server":                  "metrics-server-amd64:v0.2.1",
-		common.AddonManagerComponentName:  "kube-addon-manager-amd64:v8.6",
-		"rescheduler":                     "rescheduler:v0.3.1",
-		common.ClusterAutoscalerAddonName: "cluster-autoscaler:v1.1.2",
-		"k8s-dns-sidecar":                 "k8s-dns-sidecar-amd64:1.14.7",
-	},
-	"1.8": {
-		"addon-resizer":                  "addon-resizer:1.7",
-		"heapster":                       "heapster-amd64:v1.5.1",
-		"metrics-server":                 "metrics-server-amd64:v0.2.1",
-		"kube-dns":                       "k8s-dns-kube-dns-amd64:1.14.13",
-		common.AddonManagerComponentName: "kube-addon-manager-amd64:v8.6",
-		"dnsmasq":                        "k8s-dns-dnsmasq-nanny-amd64:1.14.8",
-		"rescheduler":                    "rescheduler:v0.3.1",
-	},
-	"1.7": {
-		"dashboard":                      "kubernetes-dashboard-amd64:v1.6.3",
-		"addon-resizer":                  "addon-resizer:1.7",
-		"heapster":                       "heapster-amd64:v1.5.1",
-		"metrics-server":                 "metrics-server-amd64:v0.2.1",
-		"kube-dns":                       "k8s-dns-kube-dns-amd64:1.14.5",
-		common.AddonManagerComponentName: "kube-addon-manager-amd64:v8.6",
-		"dnsmasq":                        "k8s-dns-dnsmasq-nanny-amd64:1.14.5",
-		"rescheduler":                    "rescheduler:v0.3.1",
-	},
-	"1.6": {
-		"dashboard":                      "kubernetes-dashboard-amd64:v1.6.3",
-		"addon-resizer":                  "addon-resizer:1.7",
-		"heapster":                       "heapster-amd64:v1.3.0",
-		"metrics-server":                 "metrics-server-amd64:v0.2.1",
-		"kube-dns":                       "k8s-dns-kube-dns-amd64:1.14.5",
-		common.AddonManagerComponentName: "kube-addon-manager-amd64:v6.5",
-		"dnsmasq":                        "k8s-dns-dnsmasq-nanny-amd64:1.14.5",
-		"rescheduler":                    "rescheduler:v0.3.1",
+var kubernetesImageBaseDefaultImages = map[string]map[string]string{
```

Comment:
These are the set of images that have been wired up to work in the `kubernetesImageBase` + `<string suffix>`.